rules/crowdstrike_rules/crowdstrike_suspicious_cron.py

```diff
@@ -0,0 +1,58 @@
+import shlex
+from fnmatch import fnmatch
+
+from panther_base_helpers import (
+    crowdstrike_detection_alert_context,
+    deep_get,
+    filter_crowdstrike_fdr_event_type,
+    get_crowdstrike_field,
+)
+
+SUSPICIOUS_CRON_CMD_ARGS = {
+    # Running in unexpected locations
+    "/tmp/*",  # nosec
+    # Reaching out to the internet
+    "curl",
+    "dig",
+    "http?://*",
+    "nc",
+    "wget",
+}
+
+SUSPICIOUS_CRON_CMDS = {
+    # Passing arguments into /bin/sh
+    "*|*sh",
+    "*sh -c *",
+}
+
+
+def suspicious_cmd_pairs(command):
```

Comment:
I recommend replacing this function with [pattern_match_list()](https://github.com/panther-labs/panther-analysis/blob/master/global_helpers/panther_base_helpers.py#L385-L387) from panther_base_helpers